src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -411,6 +462,11 @@ function createEdgeMiddleware(middlewareEntryPoint) {
 
 ### envGetSecret
 
+<p>
+
+**Type:** `SupportsKind`
+</p>
+
 This is a feature to allow your adapter to retrieve secrets configured by users in `env.schema`.
 
 Enable the feature by passing any valid `SupportsKind` value to the adapter:
```

Comment:
Ah I missed that one, since the type was renamed, this should be:
```suggestion
<p>

**Type:** `AdapterSupportsKind`
</p>

This is a feature to allow your adapter to retrieve secrets configured by users in `env.schema`.

Enable the feature by passing any valid `AdapterSupportsKind` value to the adapter:
```